draft-ietf-masque-h3-datagram.md

```diff
@@ -249,23 +141,29 @@ corresponding stream.
 # Capsules {#capsule}
```

Comment:
Editorial: Not strictly related to this PR but consider renaming this section to `Capsule Protocol` and the subsection to something else (possibly `Using the Capsule Protocol")

Reply:
Let's tackle that post design team